### ISSUE TITLE
One question about validation loss

### ISSUE THREAD

**Issue by KYuuto1006** [sail-sg/regmix#12]
Hi, thank you for your great work!
I have one question about the trend of validation losses. I am using the mixing ratios in 
``` mixture_config/config_1b/regmix.yaml ```
to train the 1B model on ```sail/regmix-data-sample```, but the trend for validation loss seems weird as shown in the figure below. 
<img width="590" height="293" alt="Image" src="https://github.com/user-attachments/assets/5524dae5-145d-44d7-93fa-81356bd4c2b6" />
Should I use the full dataset to reproduce the validation loss figure in the paper? This optimal ratio does not work for the sampled dataset? 
Waiting for your reply. Thanks.

**Comment by SivilTaram** [comment]
Hey @KYuuto1006 ! Thanks for your feedback, but you should not train 1B models on the sample data :( The sample data is just for the proxy model training, and you can switch to `sail/regmix-data` for training.
